plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt.go

```diff
@@ -50,13 +52,974 @@ type CiscoTelemetryMDT struct {
+
+	return nil
+}
+
+func (c *CiscoTelemetryMDT) InitCh() error {
+	key := "sys/ch"
+	c.nxpathMap[key] = make(map[string]string, 10)
+	c.nxpathMap[key]["fanName"] = "string"
+	c.nxpathMap[key]["typeCordConnected"] = "string"
+	c.nxpathMap[key]["vendor"] = "string"
+	c.nxpathMap[key]["model"] = "string"
+	c.nxpathMap[key]["rev"] = "string"
+	c.nxpathMap[key]["vdrId"] = "string"
+	c.nxpathMap[key]["hardwareAlarm"] = "string"
+	c.nxpathMap[key]["unit"] = "string"
+	c.nxpathMap[key]["hwVer"] = "string"
+
+	return nil
+}
+
+func (c *CiscoTelemetryMDT) InitIntf() error {
+	key := "sys/intf"
+	c.nxpathMap[key] = make(map[string]string, 10)
+	c.nxpathMap[key]["descr"] = "string"
+	c.nxpathMap[key]["name"] = "string"
+	c.nxpathMap[key]["lastStCause"] = "string"
+	c.nxpathMap[key]["description"] = "string"
+	c.nxpathMap[key]["unit"] = "string"
+	c.nxpathMap[key]["operFECMode"] = "string"
+	c.nxpathMap[key]["operBitset"] = "string"
+	c.nxpathMap[key]["mdix"] = "string"
+
+	return nil
+}
+
+func (c *CiscoTelemetryMDT) InitProcsys() error {
+	key := "sys/procsys"
+	c.nxpathMap[key] = make(map[string]string, 10)
+	c.nxpathMap[key]["name"] = "string"
+	c.nxpathMap[key]["id"] = "string"
+	c.nxpathMap[key]["upTs"] = "string"
+	c.nxpathMap[key]["interval"] = "string"
+	c.nxpathMap[key]["memstatus"] = "string"
+
+	return nil
+}
+
+func (c *CiscoTelemetryMDT) InitProc() error {
+	key := "sys/proc"
+	c.nxpathMap[key] = make(map[string]string, 2)
+	c.nxpathMap[key]["processName"] = "string"
+	c.nxpathMap[key]["procArg"] = "string"
+
+	return nil
+}
+
+func (c *CiscoTelemetryMDT) InitBfd() error {
+	key := "sys/bfd/inst"
+	c.nxpathMap[key] = make(map[string]string, 4)
+	c.nxpathMap[key]["descr"] = "string"
+	c.nxpathMap[key]["vrfName"] = "string"
+	c.nxpathMap[key]["name"] = "string"
+	c.nxpathMap[key]["name"] = "string"
+
+	return nil
+}
+
+func (c *CiscoTelemetryMDT) InitLldp() error {
+	key := "sys/lldp"
+	c.nxpathMap[key] = make(map[string]string, 7)
+	c.nxpathMap[key]["sysDesc"] = "string"
+	c.nxpathMap[key]["portDesc"] = "string"
+	c.nxpathMap[key]["portIdV"] = "string"
+	c.nxpathMap[key]["chassisIdV"] = "string"
+	c.nxpathMap[key]["sysName"] = "string"
+	c.nxpathMap[key]["name"] = "string"
+	c.nxpathMap[key]["id"] = "string"
+
+	return nil
+}
+
+func (c *CiscoTelemetryMDT) InitDb() error {
+	c.nxpathMap = make(map[string]map[string]string, 200)
+
+	c.InitPower()
+	c.InitMemPhys()
+	c.InitBgpV4()
+	c.InitCpu()
+	c.InitResources()
+	c.InitPtpCorrection()
+	c.InitTrans()
+	c.InitIgmp()
+	c.InitVrfAll()
+	c.InitIgmpSnoop()
+	c.InitIgmpSnoopGroups()
+	c.InitIgmpSnoopGroupDetails()
+	c.InitIgmpSnoopGroupsSumm()
+	c.InitMrouter()
+	c.InitSnoopStats()
+	c.InitPimInterface()
+	c.InitPimNeigh()
+	c.InitPimRoute()
+	c.InitPimRp()
+	c.InitPimStats()
+	c.InitIntfBrief()
+	c.InitPimVrf()
+	c.InitIpMroute()
+	c.InitIpv6Mroute()
+	c.InitVpc()
+	c.InitBgp()
+	c.InitCh()
+	c.InitIntf()
+	c.InitProcsys()
+	c.InitProc()
+	c.InitBfd()
+	c.InitLldp()
```

Comment:
As seemingly those functions cannot error remove the `error` from the function signature of all those functions to indicate this fact.